src/static/js/modules/alert.js

```diff
@@ -0,0 +1,76 @@
+/* ==========================================================================
+   Alert Initialization
+   Used on at least `/the-bureau/leadership-calendar/`.
+   ========================================================================== */
+
+'use strict';
+
+var $ = require( 'jquery' );
+
+function init() {
+	// Create the jQuery Alert plugin.
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/dac47241de99a43a013ef10f38e582bb8fdcdaed